### PR TITLE
Added metric type, updated readme

### PR DESCRIPTION
**Pull request by daria-lewandowska**
Fixed https://github.com/intelsdi-x/snap-plugin-collector-mysql/issues/13

Summary of changes:
 -Added new column "type" to METRIC.md, 
 -Update README.md

### CODE REVIEW

**Comment by marcintao** [comment]
Great, that's ready for merge. LGTM.

File: METRICS.md [hunk 1]
```diff
@@ -3,128 +3,128 @@
 ## Collected Metrics
 This plugin has the ability to gather the following metrics:
```
Comment (andrzej-k) [inline METRICS.md]: Please explain metric types: gauge, derive and counter.

Comment (daria-lewandowska) [inline METRICS.md]: Done

**Comment by andrzej-k** [comment]
LGTM

**Pull request merged**